lib/rake/build.rake

```diff
@@ -44,6 +45,23 @@ namespace :build do
     end
   end
 
+  desc 'Starts Active Job workers, restarts existing workers (if any) in a rolling fashion to avoid downtime'
+  timed_task_with_logging :start_active_job_workers, [:n_workers, :n_batches] do |_t, args|
+    if args[:n_workers]
+      # Manual invocation like `rake 'build:start_active_job_workers[20]'` or with `rake 'build:start_active_job_workers[20,3]'`
+      # specifying n_workers and optionally n_batches.
+      n_workers = args[:n_workers].to_i
+      n_batches = args[:n_batches] ? args[:n_batches].to_i : 3
+      Cdo::ActiveJobBackend.restart_workers(n_workers_to_start: n_workers, rolling_restart_in_n_batches: n_batches)
+    elsif rack_env?(:production)
+      Cdo::ActiveJobBackend.restart_workers(n_workers_to_start: 150, rolling_restart_in_n_batches: 5)
+    elsif CDO.test_system?
+      Cdo::ActiveJobBackend.restart_workers(n_workers_to_start: 10, rolling_restart_in_n_batches: 2)
+    else
+      Cdo::ActiveJobBackend.restart_workers(n_workers_to_start: 1)
```

Comment:
not blocking for this PR, but we may want to follow up by extracting CDO variables for the hard-coded numbers:
* CDO.num_delayed_job_workers
* CDO.num_delayed_job_batches

Reply:
Implemented as:
```
CDO.active_job_backend_n_workers_to_start
CDO.active_job_backend_rolling_restart_in_n_batches
```

Kind of a mouthful, but at least ties back to which code uses it.

@davidsbailey Would you take a look at the setup for these in config.yml.erb, test.yml.erb, etc? Seems to work locally.